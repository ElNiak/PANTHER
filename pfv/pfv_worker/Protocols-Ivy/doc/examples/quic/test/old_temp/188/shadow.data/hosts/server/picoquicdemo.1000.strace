00:00:00.000000000 [tid 1000] shadow_init_memory_manager(...) = 0
000000000000000000 [tid 1000] gettimeofday(...) = 0
000000000000000000 [tid 1000] time(...) = 946684800
00:00:00.000000000 [tid 1000] fstat(...) = 0
000000000000001020 [tid 1000] gettimeofday(...) = 0
00:00:00.000001020 [tid 1000] openat(-100, "/tmp/QUIC-FormalVerification/tickets/tic"..., (empty), (empty)) = 3
00:00:00.000002030 [tid 1000] fstat(...) = 0
00:00:00.000002030 [tid 1000] read(3, 0x555555618fc0, 4096) = 0
00:00:00.000004030 [tid 1000] close(3) = 0
00:00:00.000004030 [tid 1000] futex(...) = 0
00:00:00.000006030 [tid 1000] futex(...) = 0
00:00:00.000006030 [tid 1000] futex(...) = 0
00:00:00.000008030 [tid 1000] futex(...) = 0
00:00:00.000008030 [tid 1000] brk(0x555555653000) = 1432694784
00:00:00.000010030 [tid 1000] futex(...) = 0
00:00:00.000010030 [tid 1000] futex(...) = 0
00:00:00.000012030 [tid 1000] futex(...) = 0
00:00:00.000012030 [tid 1000] futex(...) = 0
00:00:00.000014030 [tid 1000] futex(...) = 0
00:00:00.000014030 [tid 1000] futex(...) = 0
00:00:00.000016030 [tid 1000] futex(...) = 0
00:00:00.000016030 [tid 1000] futex(...) = 0
00:00:00.000018030 [tid 1000] futex(...) = 0
00:00:00.000018030 [tid 1000] openat(-100, "/tmp/QUIC-FormalVerification/quic-implem"..., (empty), (empty)) = 3
00:00:00.000020030 [tid 1000] fstat(...) = 0
00:00:00.000020030 [tid 1000] read(3, 0x55555563be40, 4096) = 1159
00:00:00.000022030 [tid 1000] read(3, 0x55555563be40, 4096) = 0
00:00:00.000022030 [tid 1000] close(3) = 0
00:00:00.000024030 [tid 1000] openat(-100, "/tmp/QUIC-FormalVerification/quic-implem"..., (empty), (empty)) = 3
00:00:00.000024030 [tid 1000] fstat(...) = 0
00:00:00.000026030 [tid 1000] read(3, 0x55555563c100, 4096) = 1704
00:00:00.000026030 [tid 1000] close(3) = 0
00:00:00.000028030 [tid 1000] getrandom(0x7fffffffdf90, 32, 0) = 32
00:00:00.000028030 [tid 1000] getrandom(0x7fffffffe110, 24, 0) = 24
00:00:00.000030030 [tid 1000] getrandom(0x555555618d28, 64, 0) = 64
000000000000030030 [tid 1000] gettimeofday(...) = 0
00:00:00.000031040 [tid 1000] openat(-100, "demo_token_store.bin", (empty), (empty)) = -2 (ENOENT)
00:00:00.000031040 [tid 1000] write(2, "No token file present. Will create one a"..., 66) = 66
000000000000033040 [tid 1000] gettimeofday(...) = 0
00:00:00.000033040 [tid 1000] socket(Inet, 2, 17) = 3
00:00:00.000034050 [tid 1000] setsockopt(3, 0, 1, 0x7fffffffd714, 4) = 0
00:00:00.000034050 [tid 1000] setsockopt(3, 0, 13, 0x7fffffffd714, 4) = 0
00:00:00.000036050 [tid 1000] setsockopt(3, 0, 8, 0x7fffffffd734, 4) = 0
00:00:00.000036050 [tid 1000] bind(3, 11.0.0.2:4443, 16) = 0
00:00:00.000038050 [tid 1000] getsockname(3, 0x7fffffffd790, 128 (0x7fffffffd734)) = 0
00:00:01.030784010 [tid 1000] select(...) = 1
00:00:01.030784010 [tid 1000] recvmsg(3, msghdr { msg_name: SockaddrStorage { len: 128, family: None, .. } (0x7fffffffd9d0), msg_namelen: 128, msg_iov: 0x7fffffffd320, msg_iovlen: 1, msg_control: 0x7fffffffd3f0, msg_controllen: 1024, msg_flags: (empty) } (0x7fffffffd330), (empty)) = 1232
000000001030786010 [tid 1000] gettimeofday(...) = 0
00:00:01.030786010 [tid 1000] getrandom(0x55555563c770, 8, 0) = 8
00:00:01.030787020 [tid 1000] getrandom(0x55555564f820, 2, 0) = 2
000000001030787020 [tid 1000] gettimeofday(...) = 0
00:00:01.030788030 [tid 1000] openat(-100, "/qlog.bin/0000000000000001.910e.server.l"..., O_RDONLY | O_WRONLY | O_CREAT | O_TRUNC, S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH) = -2 (ENOENT)
00:00:01.030788030 [tid 1000] write(1, "Starting Picoquic server (v1.01a) on por"..., 712) = 712
00:00:01.030790030 [tid 1000] brk(0x555555674000) = 1432829952
00:00:01.030790030 [tid 1000] write(1, "0000000000000001: Sending packet type: 2"..., 385) = 385
00:00:01.030792030 [tid 1000] select(...) = 0
000000001030792030 [tid 1000] gettimeofday(...) = 0
000000001030793040 [tid 1000] gettimeofday(...) = 0
00:00:01.030793040 [tid 1000] sendmsg(3, msghdr { msg_name: 11.0.0.1:4987 (0x7fffffffdad0), msg_namelen: 16, msg_iov: 0x7fffffffd3c0, msg_iovlen: 1, msg_control: 0x0, msg_controllen: 0, msg_flags: (empty) } (0x7fffffffd3d0), (empty)) = 51
00:00:11.030794050 [tid 1000] select(...) = 0
000000011030794050 [tid 1000] gettimeofday(...) = 0
00:00:21.030795060 [tid 1000] select(...) = 0
000000021030795060 [tid 1000] gettimeofday(...) = 0
00:00:31.030796070 [tid 1000] select(...) = 0
000000031030796070 [tid 1000] gettimeofday(...) = 0
00:00:41.030797080 [tid 1000] select(...) = 0
000000041030797080 [tid 1000] gettimeofday(...) = 0
00:00:51.030798090 [tid 1000] select(...) = 0
000000051030798090 [tid 1000] gettimeofday(...) = 0
00:01:01.030799100 [tid 1000] select(...) = 0
000000061030799100 [tid 1000] gettimeofday(...) = 0
00:01:11.030800110 [tid 1000] select(...) = 0
000000071030800110 [tid 1000] gettimeofday(...) = 0
00:01:21.030801120 [tid 1000] select(...) = 0
000000081030801120 [tid 1000] gettimeofday(...) = 0
00:01:31.030802130 [tid 1000] select(...) = 0
000000091030802130 [tid 1000] gettimeofday(...) = 0
00:01:41.030803140 [tid 1000] select(...) = 0
000000101030803140 [tid 1000] gettimeofday(...) = 0
00:01:51.030804150 [tid 1000] select(...) = 0
000000111030804150 [tid 1000] gettimeofday(...) = 0
00:02:01.030805160 [tid 1000] select(...) = 0
000000121030805160 [tid 1000] gettimeofday(...) = 0
00:02:11.030806170 [tid 1000] select(...) = 0
000000131030806170 [tid 1000] gettimeofday(...) = 0
00:02:21.030807180 [tid 1000] select(...) = 0
000000141030807180 [tid 1000] gettimeofday(...) = 0
00:02:31.030808190 [tid 1000] select(...) = 0
000000151030808190 [tid 1000] gettimeofday(...) = 0
00:02:41.030809200 [tid 1000] select(...) = 0
000000161030809200 [tid 1000] gettimeofday(...) = 0
00:02:51.030810210 [tid 1000] select(...) = 0
000000171030810210 [tid 1000] gettimeofday(...) = 0
00:03:01.030811220 [tid 1000] select(...) = 0
000000181030811220 [tid 1000] gettimeofday(...) = 0
00:03:11.030812230 [tid 1000] select(...) = 0
000000191030812230 [tid 1000] gettimeofday(...) = 0
00:03:21.030813240 [tid 1000] select(...) = 0
000000201030813240 [tid 1000] gettimeofday(...) = 0
00:03:31.030814250 [tid 1000] select(...) = 0
000000211030814250 [tid 1000] gettimeofday(...) = 0
00:03:41.030815260 [tid 1000] select(...) = 0
000000221030815260 [tid 1000] gettimeofday(...) = 0
00:03:51.030816270 [tid 1000] select(...) = 0
000000231030816270 [tid 1000] gettimeofday(...) = 0
00:04:01.030817280 [tid 1000] select(...) = 0
000000241030817280 [tid 1000] gettimeofday(...) = 0
00:04:11.030818290 [tid 1000] select(...) = 0
000000251030818290 [tid 1000] gettimeofday(...) = 0
00:04:21.030819300 [tid 1000] select(...) = 0
000000261030819300 [tid 1000] gettimeofday(...) = 0
00:04:31.030820310 [tid 1000] select(...) = 0
000000271030820310 [tid 1000] gettimeofday(...) = 0
00:04:41.030821320 [tid 1000] select(...) = 0
000000281030821320 [tid 1000] gettimeofday(...) = 0
00:04:51.030822330 [tid 1000] select(...) = 0
000000291030822330 [tid 1000] gettimeofday(...) = 0
