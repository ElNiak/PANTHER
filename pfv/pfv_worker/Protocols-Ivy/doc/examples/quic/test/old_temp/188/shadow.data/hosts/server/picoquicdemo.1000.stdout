Starting Picoquic server (v1.01a) on port 4443, server name = ::, just_once = 0, do_retry = 0
Waiting for packets.
0000000000000001: Receiving 1232 bytes from 11.0.0.1:4987 at T=0.000000 (35d013b479a82)
0000000000000001: Receiving packet type: 2 (initial), S0, Q1, Version ff00001d,
0000000000000001:     <0000000000000001>, <0000000000000000>, Seq: 1, pl: 1206
0000000000000001:     Token length: 0
0000000000000001:     Decrypted 1186 bytes
0000000000000001:     Crypto HS frame, offset 0, length 215: 0000000000000000...
0000000000000001:     padding, 967 bytes

0000000000000001: Picotls returns error: 40 (0x28)
0000000000000001: Protocol error 0x128
0000000000000001: Server CB, Stream: 0, 0 bytes, fin=5

0000000000000001: Sending packet type: 2 (initial), S0, Q1, Version ff00001d,
0000000000000001:     <0000000000000000>, <7a4f53636fb80cb0>, Seq: 0, pl: 25
0000000000000001:     Token length: 0
0000000000000001:     Prepared 5 bytes
0000000000000001:     connection_close, Error 0x0128, Reason length 0

0000000000000001: Sending 51 bytes to 11.0.0.1:4987 at T=0.000000 (35d013b479a82)
